index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fitness</title>
+    <link rel="stylesheet" href="main.css" />
+    <script
+      src="https://kit.fontawesome.com/667d7efdcc.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <div class="desk_nav">
+        <div class="list-1">
+          <ul>
+            <i class="fa-brands fa-facebook"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <li><a href="#">Logout</a></li>
+            <li><a href="#">Mypage</a></li>
+            <li><a href="#">Somali</a></li>
+          </ul>
+        </div>
+        <nav class="nav">
+          <div lgo>
+            <a href="index.html">
+              <img src="./img/logo.png" alt="" class="logo"
+            /></a>
+          </div>
+          <ul>
+            <a href="./about.html">About</a>
+            <a href="#">Program</a>
+            <a href="#">Join</a>
+            <a href="#">Sponsor</a>
+            <a href="#">Updates</a>
+            <h4 class="list-h4">BB compaign</h4>
+          </ul>
+        </nav>
```

Comment:
- [ ] Make sure you fixed the small bug we found in your `nav` bar. Currently, we can't click any button you may check your `z-index`. Kindly, fix it 👍 
![Capture6](https://user-images.githubusercontent.com/57408419/234559809-90426791-5711-4e16-8e53-5d725812f550.PNG)